config/api.yml

```diff
@@ -1124,10 +1128,6 @@
       :get:
       - :name: show
       - :identifier: miq_cloud_networks_view
-    :load_balancers_subcollection_actions:
-      :get:
-      - :name: show
-        :identifier: load_balancer_show
```

Comment:
We don't need that granularity right now - the identifiers are the same and based on load balancers, not the parent collection. If that changes they can always be broken out again - but I saw an opportunity to consolidate and simplify here.